Helm mtoffl01 (#10947)
DataDog/documentation: content/en/agent/kubernetes/log.md

[1]: /integrations/consul/
[2]: /agent/guide/agent-commands/
{{% /tab %}}
{{% tab "Helm" %}}

> You can customize logs collection per integration within `confd`. This method mounts the desired configuration onto the agent container.

Reviewer: ```suggestion
You can customize logs collection per integration within `confd`. This method mounts the desired configuration onto the Agent container.
```

Unlike auto-conf files, **key-value stores may use the short OR long image name as container identifiers**, e.g. `redis` OR `redis:latest`.

{{% /tab %}}
{{% tab "Helm" %}}

> The following configuration defines the integration template for `redis` containers with the `source` and `service` attributes for collecting logs:

Reviewer: ```suggestion
The following configuration defines the integration template for Redis containers with the `source` and `service` attributes for collecting logs:
```

  ```yaml
  confd:
    redis.yaml: |-
      ad_identifiers:
        - redis
      logs:
        - source: redis
        - service: redis
  ```

> Note that the above configuration would collect only logs from this integration. If you are already collecting other data from the Redis integration, you can append the `logs` section to your existing configuration.

Reviewer: ```suggestion
**Note**: The above configuration collects only logs from this integration. If you are already collecting other data from the Redis integration, you can append the `logs` section to your existing configuration.
```
